bin/rule_parser.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+'''Copyright (C) 2017 Collabora
```

Comment:
Actually that file being there was a mistake, I deleted it in my main branch but I forgot to remove it from git.